spectre-meltdown-checker.sh

```diff
@@ -4542,16 +4552,12 @@ check_mds_bsd()
 			kernel_md_clear=0
 		fi
 	else
-		if command -v "strings" >/dev/null 2>&1; then
-			if strings /boot/kernel/kernel | grep -Fq hw.mds_disable; then
-				pstatus green YES
-				kernel_md_clear=1
-			else
-				kernel_md_clear=0
-				pstatus yellow NO
-			fi
+		if grep -Fq hw.mds_disable /boot/kernel/kernel; then
```

Comment:
better using $opt_kernel here instead of hardcoding the path? (even if AFAIK under FreeBSD the location is extremely standard, but using $opt_kernel matches line 1882)

Reply:
Yes we can re-work this block, just need to easily and reliably detect that a FreeBSD kernel is running. I did not want to blindly set $opt_kernel without checking that kernel file first exists.

The enclosing block comment said:
`if we didn't find a kernel, default to guessing`